### ISSUE TITLE
testMath fails on Pangea3

### ISSUE THREAD

**Issue by XL64** [GEOSX/GEOS#1960]
**Describe the bug**
teshMath fails on Pangea3 because 2Pi != 0
Maybe adding a modulo would help

**To Reproduce**
Build and run tests on Pangea3 using host-config from repository

**Expected behavior**
Test should pass.

**Screenshots**
```
/appli_RD/LACOSTE/GEOSX/GEOSX2/src/coreComponents/LvArray/unitTests/testMath.cpp:569: Failure
Expected: (maxDiff.get()) < (3 * float( NumericLimitsNC< __half >{}.epsilon )), actual: 6.28222 vs 0.00292969
Max difference          is 6.28221765e+00
Max relative difference is 1.24008826e-03
[  FAILED  ] TestHalfMath.atan2HalfAccuracy (47 ms)
[ RUN      ] TestHalfMath.atan2Half2Accuracy
/appli_RD/LACOSTE/GEOSX/GEOSX2/src/coreComponents/LvArray/unitTests/testMath.cpp:627: Failure
Expected: (maxDiff.get()) < (3 * float( NumericLimitsNC< __half >{}.epsilon )), actual: 6.28222 vs 0.00292969
Max difference          is 6.28221774e+00
Max relative difference is 1.50966642e-03
[  FAILED  ] TestHalfMath.atan2Half2Accuracy (39 ms)
[----------] 6 tests from TestHalfMath (139 ms total)

[----------] Global test environment tear-down
[==========] 102 tests from 13 test suites ran. (421 ms total)
[  PASSED  ] 100 tests.
[  FAILED  ] 2 tests, listed below:
[  FAILED  ] TestHalfMath.atan2HalfAccuracy
[  FAILED  ] TestHalfMath.atan2Half2Accuracy
```

**Platform (please complete the following information):**
- Machine Pangea3
- Compiler: gcc 8.4.1
- GEOSX Version develop

**Comment by TotoGaz** [comment]
Is this still relevant?
More recent comment https://github.com/GEOSX/GEOSX/issues/2031#issuecomment-1240866437 and first tick box in issue https://github.com/GEOSX/GEOSX/issues/2031 make me think it's OK now?

**Comment by XL64** [comment]
This was solved in #2030

**Comment by XL64** [comment]
This was solved in https://github.com/GEOSX/GEOS/pull/2030
